app/package.json

```diff
@@ -8,6 +8,7 @@
     "start": "BROWSER=none react-scripts start",
     "develop": "REACT_APP_USE_SAMPLE_DATA=true yarn start",
     "build": "react-scripts build",
+    "postbuild": "echo -n '# Keep directory in git.' > build/.gitkeep",
```

Comment:
so the front end build deletes the directory?

Reply:
Yes, the react build scripts clear out the whole build folder. So this is a bit hacky but IMO gets the job done.